src/ImpactEvaluator.sol

```diff
@@ -52,7 +52,7 @@ contract ImpactEvaluator is AccessControl {
     }
 
     function maybeAdvanceRound() private {
-        uint currentRoundEnd = openRounds[currentRoundIndex()].end;
+        uint currentRoundEnd = getOpenRound(currentRoundIndex()).end;
```

Comment:
since we shrink the array, we can't use indexes but need the utility function instead

Reply:
Do you have any concerns about the gas costs of iterating over all open rounds twice?

First time, we call `currentRoundIndex()` here.
Second time, we call `currentRoundIndeX()` from `advanceRound()`

Reply:
Good catch 👀  This has been refactored already in #main :) We're storing `currentRoundIndex` in a variable now, no need to calculate it

https://github.com/Meridian-IE/impact-evaluator/blob/d4a8159645e0dc781b2a05c5789dba10d53dcfaf/src/ImpactEvaluator.sol#L65-L67

https://github.com/Meridian-IE/impact-evaluator/blob/d4a8159645e0dc781b2a05c5789dba10d53dcfaf/src/ImpactEvaluator.sol#L40-L49